charts/datahub/templates/postgresql-setup-job.yml

```diff
@@ -55,7 +55,7 @@ spec:
                   name: "{{ .Values.global.sql.datasource.password.secretRef }}"
                   key: "{{ .Values.global.sql.datasource.password.secretKey }}"
             - name: POSTGRES_HOST
-              value: {{ .Values.global.sql.datasource.hostForpostgresqlClient | quote }}
+              value: {{ .Values.global.sql.datasource.hostForPostgresqlClient | quote }}
```

Comment:
Please discard this change, the Postgres is lower case, this is a breaking change. Please also update the docs accordingly.

Reply:
Good point, done